CovertInfiltration/Src/CovertInfiltration/Classes/CI_UIStrategyMapItem_CovertAction.uc

```diff
@@ -69,17 +76,32 @@ function UpdateFromGeoscapeEntity(const out XComGameState_GeoscapeEntity Geoscap
 	}	
 }
 
-simulated function UpdateLaunchedActionBox(XComGameState_CovertAction CovertAction)
+simulated function UpdateOverinfiltratingBox(XComGameState_MissionSiteInfiltration MissionSite)
+{
+	local XComGameState_CovertAction CovertAction;
+	
+	CovertAction = XComGameState_CovertAction(`XCOMHISTORY.GetGameStateForObjectID(MissionSite.CorrespondingActionRef.ObjectID));
```

Comment:
This will fail if the player has run (another) mission since start of over infiltration phase